src/main/java/frc/robot/subsystems/VisionSubsystem.java

```diff
@@ -24,44 +13,30 @@ public double getY() {
         return LimelightHelpers.getTY(VisionConstants.LIMELIGHT_NAME);
     }
 
-    public double[] getPose() {
-        return LimelightHelpers.getBotPose(VisionConstants.LIMELIGHT_NAME);
-    }
-
-    public Pose3d getPose3d() {
-        return LimelightHelpers.getBotPose3d(VisionConstants.LIMELIGHT_NAME);
-    }
-
-    public Pose2d getPose2d() {
-        return LimelightHelpers.getBotPose2d(VisionConstants.LIMELIGHT_NAME);
-    }
-
-
-    public double getYaw() {
-        return getPose()[5];
+    public LimelightHelpers.Results getTargetingResults() {
+        return LimelightHelpers.getLatestResults(VisionConstants.LIMELIGHT_NAME).targetingResults;
     }
 
-    public double getPitch() {
-        return getPose()[4];
+    public Command ledOff() {
+        return runOnce(() -> LimelightHelpers.setLEDMode_ForceOff(VisionConstants.LIMELIGHT_NAME));
     }
 
-    public double getID() {
-        return LimelightHelpers.getFiducialID(VisionConstants.LIMELIGHT_NAME);
+    public Command ledOn() {
+        return runOnce(() -> LimelightHelpers.setLEDMode_ForceOn(VisionConstants.LIMELIGHT_NAME));
     }
 
-    public void setPipeline(int num) {
-        LimelightHelpers.setPipelineIndex(VisionConstants.LIMELIGHT_NAME, num);
-    }
+    public Command blinkLimelight() {
+        SequentialCommandGroup blinkCommands = new SequentialCommandGroup();
+        blinkCommands.addRequirements(this);
 
-    public LimelightHelpers.Results getTargetingResults() {
-        return LimelightHelpers.getLatestResults(VisionConstants.LIMELIGHT_NAME).targetingResults;
-    }
-
-    public void ledOff() {
-        LimelightHelpers.setLEDMode_ForceOff(VisionConstants.LIMELIGHT_NAME);
-    }
+        for (int i = 0; i < VisionConstants.NUM_BLINKS; i++) {
+            blinkCommands.addCommands(Commands.sequence(
+                    ledOn(),
+                    new WaitCommand(VisionConstants.BLINK_DELAY),
```

Comment:
Opinionated, but I personally prefer:

```suggestion
Commands.waitSeconds(VisionConstants.BLINK_DELAY)
```

I think it's just a bit more readable

Reply:
Can even do a static import 

```java
import static edu.wpi.first.wpilibj2.command.Commands.waitSeconds;
```

Then it's just

```java
waitSeconds(...)
```